org.gnome.Evince.json

```diff
@@ -13,9 +13,14 @@
         "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
         "--metadata=X-DConf=migrate-path=/org/gnome/evince/",
         "--env=GDK_PIXBUF_MODULE_FILE=/app/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--talk-name=org.gtk.vfs",
         "--talk-name=org.gtk.vfs.*",
         "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.freedesktop.FileManager1",
```

Comment:
Please justify this addition with a reference to the upstream commit.

Reply:
https://gitlab.gnome.org/GNOME/evince/-/commit/5bab224509ad89937af46c53129d481e75065c7d

>  Make "Open Containing Folder" work in flatpak
> 
> It doesn't seem possible to find the default application for files of
> type "inode/directory" from inside the sandbox. Try the
> org.freedesktop.FileManager1 dbus interface should this lookup fail.

Reply:
**in the commit message**

Reply:
Sorry, what do you mean? I don't see any words saying this shouldn't be done?

Reply:
I mean that you should put the justification in the commit message, not in this PR. We need the justification to be available when somebody looks through the changes made using "git log" or equivalent.